### DISCUSSION TITLE
[CUSA28863] - Elden Ring 1.01

### DISCUSSION THREAD

**Discussion by rassha** [GoldHEN/GoldHEN_Cheat_Repository#37] [Cheat requests]
I would be happy if there are cheats for this game.

Inf mana, health, god mode, (everything possible)

**Comment by rassha** [comment]
Version is 1.01
But, it's actually 1.00, backport update it to 1.01

**Comment by Samanmansouri7** [comment]
Please chate elden ring 1.01

**Comment by kmeps4** [comment]
https://github.com/GoldHEN/GoldHEN_Cheat_Repository/discussions/42#discussion-4222094

**Comment by RagztheKing510** [comment]
Where the cheats for version 1.00??
